crates/sui-framework/packages/sui-framework/sources/test/test_scenario.move

```diff
@@ -353,9 +365,45 @@ module sui::test_scenario {
         sui::transfer::share_object_impl(t)
     }
 
+    /// Return the IDs of the child objects that `parent` owns.
+    public fun child_object_ids_for_parent_object_id<T: key>(parent: ID): vector<ID> {
```

Comment:
Is there potential for confusion by using the child/parent nomenclature to refer to objects sent to other objects, seeing as it is also used for the dynamic field internals? Not entirely sure what the better option is here (... `owner` and `receivable`?)

Reply:
More than happy to bikeshed on the nomenclature here -- I've always referred to them as/the best I've been able to come up with is: `parent`/`child`, or `object`/`object-owned-object` (although I tend to not use this last one since it's a bit long...). 

`owner`/`receivable` might be nice since with the addition of private receive rules for `key`-only objects, you don't have "ownership" over the object in the sense that you're guaranteed access to it. + `receivable_ids_for_owner_id` has a nice ring to it.

Reply:
+1 to no parent/child nomenclature. It is really an implementation detail from the Move perspective.